### ISSUE TITLE
Accessibility Issues - Ecommerce

### ISSUE THREAD

**Issue by Jowens96** [Jowens96/ecommerce-pattern-library#2]
@thomasjbradley

- As it stands there are no alt attributes or ARIA labels applied to any images.
- Links are not obviously displayed. They could use an underline.
- Colour contrast is probably too low.

**Comment by thomasjbradley** [comment]
Thanks!
